packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -557,19 +623,22 @@ export class AccountHandler {
       emailVerified: false,
       kA,
       wrapWrapKb,
+      wrapWrapKbVersion2: null,
       authSalt,
       verifierVersion: this.config.verifierVersion,
       verifyHash: Buffer.alloc(32).toString('hex'),
+      verifyHashVersion2: null,
       verifierSetAt: 0,
       locale: request.app.acceptLanguage,
+      clientSalt: null,
     });
 
     const access = await generateAccessToken({
       clientId: client.id,
       name: client.name,
       canGrant: client.canGrant,
       publicClient: client.publicClient,
-      userId: account.uid,
+      userId: uuidTransformer.to(account.uid),
```

Comment:
@chenba Also calling this change out, it appears this would fail test our integration without, which seems a bit suspicious...